pennylane/devices/execution_config.py

```diff
@@ -20,6 +20,22 @@
 from pennylane.workflow import SUPPORTED_INTERFACES
 
 
+@dataclass
+class MCMConfig:
+    """A class to store mid-circuit measurement configurations."""
+
+    mcm_method: Optional[str] = None
+    """Which mid-circuit measurement strategy to use. Use ``deferred`` for the deferred
```

Comment:
What does `None` indicate? The device makes it's own decision?

Reply:
Yes. I'll add documentation.